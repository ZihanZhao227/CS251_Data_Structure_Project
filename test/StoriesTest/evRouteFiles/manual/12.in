50
0 0 0 CITY Start
1 91 287 CHARGING_STATION CHARGING_STATION_1
2 136 129 RESTAURANT RESTAURANT_2
3 210 121 RESTAURANT RESTAURANT_3
4 487 87 RESTAURANT RESTAURANT_4
5 246 428 RESTAURANT RESTAURANT_5
6 146 158 RESTAURANT RESTAURANT_6
7 216 475 RESTAURANT RESTAURANT_7
8 216 356 RESTAURANT RESTAURANT_8
9 170 156 RESTAURANT RESTAURANT_9
10 121 307 CHARGING_STATION CHARGING_STATION_10
11 364 310 CHARGING_STATION CHARGING_STATION_11
12 110 457 RESTAURANT RESTAURANT_12
13 301 329 CHARGING_STATION CHARGING_STATION_13
14 257 146 CHARGING_STATION CHARGING_STATION_14
15 328 114 CHARGING_STATION CHARGING_STATION_15
16 139 440 RESTAURANT RESTAURANT_16
17 284 482 RESTAURANT RESTAURANT_17
18 497 79 RESTAURANT RESTAURANT_18
19 242 78 CHARGING_STATION CHARGING_STATION_19
20 123 315 RESTAURANT RESTAURANT_20
21 172 196 CHARGING_STATION CHARGING_STATION_21
22 63 457 CHARGING_STATION CHARGING_STATION_22
23 402 42 RESTAURANT RESTAURANT_23
24 352 183 CHARGING_STATION CHARGING_STATION_24
25 118 56 CHARGING_STATION CHARGING_STATION_25
26 150 90 RESTAURANT RESTAURANT_26
27 33 461 RESTAURANT RESTAURANT_27
28 435 338 RESTAURANT RESTAURANT_28
29 177 33 RESTAURANT RESTAURANT_29
30 92 84 RESTAURANT RESTAURANT_30
31 296 17 RESTAURANT RESTAURANT_31
32 335 395 CHARGING_STATION CHARGING_STATION_32
33 112 362 CHARGING_STATION CHARGING_STATION_33
34 449 452 RESTAURANT RESTAURANT_34
35 121 346 RESTAURANT RESTAURANT_35
36 471 492 RESTAURANT RESTAURANT_36
37 473 428 CHARGING_STATION CHARGING_STATION_37
38 28 304 RESTAURANT RESTAURANT_38
39 320 11 RESTAURANT RESTAURANT_39
40 420 12 CHARGING_STATION CHARGING_STATION_40
41 404 290 RESTAURANT RESTAURANT_41
42 479 361 RESTAURANT RESTAURANT_42
43 444 402 CHARGING_STATION CHARGING_STATION_43
44 47 484 RESTAURANT RESTAURANT_44
45 129 135 CHARGING_STATION CHARGING_STATION_45
46 448 147 RESTAURANT RESTAURANT_46
47 159 25 RESTAURANT RESTAURANT_47
48 393 279 RESTAURANT RESTAURANT_48
49 500 500 CITY End
