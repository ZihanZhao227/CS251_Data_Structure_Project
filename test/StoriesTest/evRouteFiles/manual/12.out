0 49 100.000 -1.0
NO_PATH
